File: Data Structures & Algorithms/climbing-stairs/submission-1.java
class Solution {
    public int climbStairs(int n) {
        int[] answer = new int[n+1];
        int total = 0;



        for(int i = answer.length -1; i >= 0; i--){
            System.out.println(i);
            if (i == n){
                answer[i] = 1;
            
            }
            else if(i == n-1){
                answer[i] = 1;
            }
            else{
                answer[i] = answer[i+2] + answer[i+1];
            }
        }

        return answer[0];
    }
}

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        #k most frequent,  do we do counts?
        
        mappings = {}
        for num in nums:
            if num not in mappings:
                mappings[num] = 0
            mappings[num] +=1
        
        things = []
        for item in mappings:
            things.append((mappings[item], item))
        
        things.sort()

        answer = []
        
        for x in range(k):
            index = len(things) -1 -x
            answer.append(things[index][1])
        
        return answer


            